src/resources/dashboard.yml

```diff
@@ -12,18 +12,18 @@ dashboard:
     
     in-review:
       message_type: success
-      title: Your application is being reviewed &nbsp; &#9989; # Tick emoji
```

Comment:
The tick emoji provides a nice visual indicator of success — why do you want to remove it?

Reply:
Three reasons: it messes with the line-height, it introduces a different colour palette, and there isn't a consistent emoji-use strategy. If you'd like emoji, I'd suggest `<span class="pixel">` old school emoticons, and then I'd come up with a consistent way of using them.